poh/src/poh_recorder.rs

```diff
@@ -560,6 +648,8 @@ impl PohRecorder {
             self.tick_height += 1;
             trace!("tick_height {}", self.tick_height);
 
+            self.report_poh_timing_point();
```

Comment:
I think it'll be more accurate to move this after the sleep: https://github.com/solana-labs/solana/blob/2544aae9e9fe0d8a6d61c9044528da24467fd359/poh/src/poh_recorder.rs#L673-L687 (adjusting for faster hardware)

Reply:
i see, thanks for the explainer